### PR TITLE
 add nomics-test v3 ea

### PR DESCRIPTION
**Pull request by karen-stepanyan**

### CODE REVIEW

File: packages/sources/nomics-test/src/endpoint/filtered.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  CustomSettings: typeof customSettings
+  Provider: {
+    RequestBody: ProviderRequestBody
+    ResponseBody: ResponseSchema
+  }
+}
+
+const restEndpointTransport = new RestTransport<FilteredEndpointTypes>({
+  prepareRequest: (p, config) => {
+    const baseURL = config.API_ENDPOINT
```
Comment (alejoberardino) [inline packages/sources/nomics-test/src/endpoint/filtered.ts:55]: Seems like nomics supports batches on the base side, but not the quote. Same thing as the kaiko one, we should block this one.

**Review by alejoberardino — DISMISSED** [review]

File: packages/sources/nomics-test/src/config/index.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export const customSettings = {
+  API_KEY: {
+    description: 'An API key that can be obtained from https://p.nomics.com/pricing#free-plan',
+    type: 'string',
+    required: true,
+  },
```
Comment (boxhock) [inline packages/sources/nomics-test/src/config/index.ts:6]: Make this sensitive?

File: packages/sources/nomics-test/src/crypto-utils.ts [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  high_timestamp: string
+  '1d': {
+    volume: string
+    price_change: string
+    price_change_pct: string
+    volume_change: string
+    volume_change_pct: string
+    market_cap_change: string
+    market_cap_change_pct: string
+  }
```
Comment (boxhock) [inline packages/sources/nomics-test/src/crypto-utils.ts:55]: Can we extract this into its own type and re-use that? Assuming it's the same payload for each of these time ranges

File: packages/sources/nomics-test/src/endpoint/filtered.ts (hunk 1 above)
Comment (boxhock) [reply packages/sources/nomics-test/src/endpoint/filtered.ts:50]: Can this be unblocked now? Can we make this EA send batches on the base, like the crypto endpoint?

**Review by boxhock — CHANGES_REQUESTED** [review]

File: packages/sources/nomics-test/src/endpoint/filtered.ts (hunk 1 above)
Comment (karen-stepanyan) [reply packages/sources/nomics-test/src/endpoint/filtered.ts:50]: I don't really think `/prices/restricted` API endpoint is partially batchable. As a matter of fact i didn't even find any documentation for that. There is a `prices` API endpoint where just one ticker is allowed.

File: packages/sources/nomics-test/src/config/index.ts (hunk 2 above)
Comment (bogos) [reply packages/sources/nomics-test/src/config/index.ts]: Done

File: packages/sources/nomics-test/src/endpoint/filtered.ts (hunk 1 above)
Comment (boxhock) [reply packages/sources/nomics-test/src/endpoint/filtered.ts:50]: Should we just mark this as resolved then?

**Review by boxhock — APPROVED** [review]
LGTM

**Comment by amit-momin** [comment]
I think this is good to go but let's do one more framework bump before we merge this. The recent patches had some important fixes

**Review by amit-momin — APPROVED** [review]

**Pull request merged**